package.json

```diff
@@ -28,12 +28,13 @@
     "prepare": "yarn prettify:package && yarn build",
     "publish:packages": "lerna publish",
     "start": "concurrently \"yarn build:watch\" \"yarn storybook\"",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -s ./packages/asc-assets/lib -p 6006",
+    "storybook:build": "yarn build && build-storybook -c .storybook -o .sb -s ./packages/asc-assets/lib",
     "test": "yarn jest",
     "test:watch": "yarn test -- --watch",
     "prettify:package": "yarn prettier-package-json --write **/package.json"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
styled-components already is declared in the asc-core package.json, so perhaps this can even be removed from this package.json, as yarn already hoists all the packages to our root node_modules dir

Reply:
Might be good, yes. The less dependencies, the better.

Reply:
Not sure though, because jest also uses styled-components, react etc...